packages/common/components/blocks/header/wrapper.marko

```diff
@@ -1,4 +1,6 @@
-$ const { separatorPosition, separatorType } = input;
+$ const { separatorPosition, separatorType, date } = input;
+
+$ const dateStyle = "text-decoration: none !important; font-size: 12px; font-family: Arial, Helvetica, sans-serif; color: #ffffff; line-height: 12px; text-transform: uppercase;"
```

Comment:
I would suggest converting this to 
`
$ const dateStyle = defaultValue(input. dateStyle, "text-decoration: none !important; font-size: 12px; font-family: Arial, Helvetica, sans-serif; color: #ffffff; line-height: 12px; text-transform: uppercase;");
`
and adding "@dateStyle": "string" to the header marko.js file for this block.  It would allow you to override it in the future if need be